packages/notifications/demo/global-alert.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { GlobalAlert, Close, Paragraph, Title } from '@zendeskgarden/react-notifications';
+import { GlobalAlertStory } from './stories/GlobalAlertStory';
+
+<Meta
+  title="Packages/Notifications/GlobalAlert"
+  component={GlobalAlert}
+  subcomponents={{
+    'GlobalAlert.Anchor': GlobalAlert.Anchor,
+    'GlobalAlert.Button': GlobalAlert.Button,
+    'GlobalAlert.Close': GlobalAlert.Close,
+    'GlobalAlert.Content': GlobalAlert.Content,
+    'GlobalAlert.Title': GlobalAlert.Title
+  }}
+/>
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story
+    name="GlobalAlert"
+    args={{
+      type: 'info'
+    }}
+    argTypes={{
+      anchor: { name: 'children', type: 'string', table: { category: 'Anchor' } },
+      isExternal: { control: { type: 'boolean' }, table: { category: 'Anchor' } },
+      button: { name: 'children', type: 'string', table: { category: 'Button' } },
+      isBasic: { control: { type: 'boolean' }, table: { category: 'Button' } },
```

Comment:
I'd like to see the button control to be configured a lot closer to modal `FooterItem[]`. In other words, the data should describe `Button[]` array with the following by default:

```
[
  { text: 'Cancel', isBasic: true },
  { text: 'OK' }
]
```

This allows Storybook to push into the extents of button positioning, group styling, etc.